.github/bump_docs.sh

```diff
@@ -2,6 +2,6 @@
 set -xe
 REPO=$1
 
-LATEST_TAG=$(curl -s "https://api.github.com/repos/$REPO/releases/latest" | jq -r '.name')
+LATEST_TAG=$(curl -s "https://api.github.com/repos/$REPO/releases/latest" | jq -r '.name' | grep -oP 'v\d+\.\d+\.\d+')
```

Comment:
why not just change it from `jq -r '.name'` to `jq -r .tag_name`?

Reply:
That... is better. I started from the existing jq command without even checking what was available from the api.